magento/magento2
# PageCache/AccessList: Add CIDR support

### Description (*)

When accepting purge requests within a network, it's easier to just supply a CIDR range. 

### Related Pull Requests

None.

### Fixed Issues (if relevant)

None.

### Manual testing scenarios (*)

1. Go to **Advanced -> System -> Full Page Cache -> Varnish Configuration**
2. Configure a CIDR range like `172.16.0.1/24`
3. Value should be valid and saved

### Questions or comments


### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#39953: PageCache/AccessList: Add CIDR support
